client/lib/reader-lists-items/store.js

```diff
@@ -0,0 +1,91 @@
+// Reader Lists Items Store
+
+// External dependencies
+import { List, fromJS } from 'immutable';
+import debugModule from 'debug';
+
+// Internal dependencies
+import { action as actionTypes } from './constants';
+import { createReducerStore } from 'lib/store';
+
+const debug = debugModule( 'calypso:reader-lists-items' ); //eslint-disable-line no-unused-vars
+
+const initialState = {
```

Comment:
maybe move the fromJS here to make it really explicit that the state is an Immutable thing